backend/src/main/java/com/google/step/servlets/WebhookServlet.java

```diff
@@ -53,14 +56,15 @@
    */
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("application/json;");
+    userAuthenticationUtil.authenticate(response);
```

Comment:
just inline this since the variable name is not being re-used.

Reply:
fixed